test/extended/images/mysql_replica.go

```diff
@@ -172,14 +172,17 @@ func replicationTestFactory(oc *exutil.CLI, tc testCase) func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(len(pods.Items)).To(o.Equal(1))
 
-		g.By("after slave is scaled to 0 and then back to 4 replicas")
-		err = oc.Run("scale").Args("dc", "mysql-slave", "--replicas=0").Execute()
-		o.Expect(err).NotTo(o.HaveOccurred())
-		err = exutil.WaitUntilPodIsGone(oc.KubeREST().Pods(oc.Namespace()), pods.Items[0].Name, 1*time.Minute)
-		o.Expect(err).NotTo(o.HaveOccurred())
-		err = oc.Run("scale").Args("dc", "mysql-slave", "--replicas=4").Execute()
-		o.Expect(err).NotTo(o.HaveOccurred())
-		assertReplicationIsWorking("mysql-master-2", "mysql-slave-1", 4)
+		// NOTE: Commented out, current template does not support multiple replicas.
```

Comment:
I think we can remove it, git preserves the history. You can add TODO to add multiple replaces support to the template if it will happen in the future.